### ISSUE TITLE
loading queries

### ISSUE THREAD

**Issue by seanaguinaga** [rrdelaney/react-18-relay-nextjs-repro#1]
Hello

have you figured out a strategy for nextjs + relay with their new concurrent features for loading and preloading queries with relay?

I am confused on how someone would do this.

**Comment by seanaguinaga** [comment]
Is this really all I need to do?

https://github.com/seanaguinaga/react-18-relay-nextjs-repro

```

export default function Home() {
  let preloadedQuery = loadQuery<indexPage_indexQuery>(
    useRelayEnvironment(),
    indexPage,
    {}
  );

  return (
    <Suspense fallback="loading blocks">
      <Blocks queryReference={preloadedQuery} />
    </Suspense>
  );
}
```

**Comment by rrdelaney** [comment]
This repo was meant for reproducing an issue with Relay + Next.js + the experimental runtime, which was resolved in Relay 13.0.2. Please see https://github.com/RevereCRE/relay-nextjs for using Relay + Next.js.
